### ISSUE TITLE
test_table_to_polygon_pro fails with "column was specified that does not exist" error

### ISSUE THREAD

**Issue by mfunk** [Esri/military-tools-geoprocessing-toolbox#119]
Dialog output from '_pro' test is below. The problem is in the **PolylineToPolygon.py** supporting script tool.

This is the script that takes a set of polylines and creates polygons for each closed line. The script takes an _inputIDFieldName_ as a parameter, adds it to the _outputPolygonsFC_. However the field name doesn't seem to be added to the feature class, and when the DA cursor is called (including this field), the script fails.

```
.....TableToPointTestCase.test_table_to_point_pro
ArcPy works
.....TableToPolygonTestCase.test_table_to_polygon_pro
Spatial reference is <geoprocessing spatial reference object object at 0x000000000BB09630>
Creating output feature class...
Adding ID field ...
Opening cursors ...
ERROR: PYTHON ERRORS:
Traceback info:
  File "D:\GitHub\military-tools-geoprocessing-toolbox\toolboxes\scripts\PolylineToPolygon.py", line 49, in <module>
    for row in inRows:

Error Info:
A column was specified that does not exist.
ERROR: ArcPy ERRORS:
Executing: AddField D:\GitHub\military-tools-geoprocessing-toolbox\testdata\scratch.gdb\outputPolygons RowID "Long (large integer)" # # # # NULLABLE NON_REQUIRED #
Start Time: Tuesday, June 7, 2016 8:02:50 AM
Adding RowID to D:\GitHub\military-tools-geoprocessing-toolbox\testdata\scratch.gdb\outputPolygons...
Succeeded at Tuesday, June 7, 2016 8:02:50 AM (Elapsed Time: 0.09 seconds)

PYTHON ERRORS:
Traceback info:
  File "D:\GitHub\military-tools-geoprocessing-toolbox\toolboxes\scripts\PolylineToPolygon.py", line 49, in <module>
    for row in inRows:

Error Info:
A column was specified that does not exist.

ArcPy ERRORS:
Executing: AddField D:\GitHub\military-tools-geoprocessing-toolbox\testdata\scratch.gdb\outputPolygons RowID "Long (large integer)" # # # # NULLABLE NON_REQUIRED #
Start Time: Tuesday, June 7, 2016 8:02:50 AM
Adding RowID to D:\GitHub\military-tools-geoprocessing-toolbox\testdata\scratch.gdb\outputPolygons...
Succeeded at Tuesday, June 7, 2016 8:02:50 AM (Elapsed Time: 0.09 seconds)
```

**Comment by mfunk** [comment]
Looking at this I think it is because the "Add Unique Row ID" tool was removed. This tool added a field "RowID" that the "Polyline To Polygon" script tool used.

**Comment by mfunk** [comment]
Fixed in #146
